sdk/Trace/Event.php

```diff
@@ -13,18 +12,17 @@ class Event implements API\Event
     private $timestamp;
     private $attributes;
 
-    // todo: pick datatype for timestamp
-    public function __construct(string $name, ?API\Attributes $attributes = null, string $timestamp = null)
+    public function __construct(string $name, ?API\Attributes $attributes = null, int $timestamp = null)
     {
         if (null === $timestamp) {
-            $timestamp = (new Clock())->millitime();
+            $timestamp = (new Clock())->timestamp();
```

Comment:
I think the timestamp should be required; this way `Event` doesn't have to know anything about clocks.

Reply:
Seems like a reasonable ask.  I think that is a fair expectation.